backend/middleware/sanitize.test.ts

```
@@ -206,14 +200,10 @@
       },
     };
-    const result = sanitizeWithSchema(input, schema) as Record<string, unknown>;
-    expect(result['a']).not.toMatch(/onerror|<img/i);
-    expect((result['b'] as Record<string, unknown>)['c'] as string).not.toMatch(
-      /<svg/i
-    );
-    expect((result['b'] as Record<string, unknown>)['d'] as string).not.toMatch(
-      /javascript:/i
-    );
+    const result = sanitizeWithSchema(input, schema);
+    expect((result as any).a).not.toMatch(/onerror|<img/i);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)